routers/admin/users.go

```diff
@@ -93,7 +94,10 @@ func NewUserPost(ctx *context.Context, form auth.AdminCreateUserForm) {
 			u.LoginName = form.LoginName
 		}
 	}
-
+	if !util.CheckPasswordComplexity(form.Password) {
```

Comment:
Perhaps an admin should be able to skip password requirement checks?

Reply:
Add as a standalone option in configuration file?

Reply:
@T-M-A no need for a configuration option; either it's allowed or not. But I'm not saying that it should be allowed; I'm just opening the issue for debate here.

Reply:
Fixed.

All code move to standalone module